Gruntfile.js

```diff
@@ -1,112 +1,115 @@
+/* globals module, require */
+
+const loadGruntTasks = require('load-grunt-tasks');
 module.exports = function(grunt) {
   'use strict';
-  require('load-grunt-tasks')(grunt);
+
+  loadGruntTasks(grunt);
 
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
 
-    build_app_js: 'build/resources/main/public/js/app.js',
```

Comment:
Yes. see http://eslint.org/docs/rules/camelcase and https://github.com/airbnb/javascript#naming--camelCase